concurrency-limits-core/src/main/java/com/netflix/concurrency/limits/limit/functions/Log10RootIntFunction.java

```diff
@@ -0,0 +1,34 @@
+package com.netflix.concurrency.limits.limit.functions;
+
+import java.util.function.IntUnaryOperator;
+
+/**
+ * Function used by limiters to calculate thresholds using log10 of the current limit.
+ * Here we pre-compute the log10 of numbers up to 1000 as an optimization.
+ */
+public final class Log10RootIntFunction implements IntUnaryOperator {
```

Comment:
Do we want to keep the public no-arg constructor here or have a private no-arg constructor, given the static `create` method?

Reply:
Yeah, good call, I've made the constructor private.